tests/e2e/project.spec.ts

```diff
@@ -315,23 +327,7 @@ test("peer and branch switching", async ({ page }) => {
   }
 });
 
-test("clone modal", async ({ page }) => {
```

Comment:
Can we just `skip` this one instead of deleting it?

Reply:
I'm not deleting it, it's [here](https://github.com/radicle-dev/radicle-interface/pull/515/files/1483e5fc7093f4ca95b94548181260254d94ced6#diff-be51c4ac73028270cc9313bdb4fbbcdcfa7daef13593bae12ade1990d6f60935R241)

Reply:
Ah, right. 🤦